ts_src/psetv2/input.ts

```diff
@@ -261,7 +261,7 @@ export class PsetInput {
           input.tapLeafScript!.push({
             controlBlock,
             leafVersion,
-            script: kp.value,
+            script: kp.value.slice(0, kp.value.length - 1),
```

Comment:
slice is deprecated from node 18, best `subarray`? (it does not return a copy tho)

Reply:
also you can do subarray(0,-1) no?

Reply:
Is deprecated in nodejs std library, but lets upgrade in adedicated commit